git-providers.yaml

```diff
@@ -18,5 +23,10 @@
   apiUrl: https://try.gitea.io
   repositoryUrl: https://try.gitea.io
   type: GITEA
+  clientProperties:
+    giteaOAuthUrl: http://[YOUR-GITEA-URL]/login/oauth/authorize
+    redirectUri: http://frontend-location/fixed
   serverProperties:
     adminUser: admin
+    oauthUrl: https://github.com/login/oauth/access_token
```

Comment:
This looks weird, maybe leave empty for now?

Reply:
if you leave it empty it's no longer serializable